### DISCUSSION TITLE
Representative from Homebrew for a Python packaging discussion

### DISCUSSION THREAD

**Discussion by tgamblin** [Homebrew/discussions#4296] [Tap maintenance and Homebrew development] [answered]
Hi all: @MikeMcQuaid suggested I start a discussion on this and link him to it, so here goes.

There was recently a livestreamed panel of Python packaging people, which I guess is going to become a future edition of [Talk Python to Me](https://talkpython.fm/): https://www.youtube.com/watch?v=z50B6AmQwLw.  It's a good discussion if you keep up with happenings in the Python world.

Anyway, folks pointed out [in this Twitter thread](https://twitter.com/ralfgommers/status/1629831336474169348) that downstream consumers of packaging -- i.e., users and dsitros, were not represented, and a follow-on panel is being organized.  I'm looking for someone from `brew` who might be interested in joining to give `brew`'s perspective on Python packages.

Let me know who the right person from Homebrew is, or if y'all aren't interested, that's fine too.

**Comment by MikeMcQuaid** [comment, marked as answer]
Thanks @tgamblin! I've brought this up with all @Homebrew/maintainers in Slack and will keep you posted.

**Reply by MikeMcQuaid** [reply]
@tgamblin The following folks (@carlocab @chenrui333 @imichka @Bo98) are all up for getting involved in some form!

**Reply by tgamblin** [reply]
@MikeMcQuaid: great -- thanks!  @bskinn is organizing this so I'll let him get in touch.  Great so see so much interest.

**Reply by bskinn** [reply]
> @tgamblin The following folks (@carlocab @chenrui333 @iMichka @Bo98) are all up for getting involved in some form!

Terrific -- Carlo, Rui, Michka, Bo, thanks very much for your interest!

Please email me ASAP at bskinn@openteams.com and I'll add you to my contact group for the event planning. Please also include a good Google account to use for GDoc sharing, if different from the address you email from.

**Comment by juborazchowdhury3-cmd** [comment]
Hi
